### DISCUSSION TITLE
Is it possible to store all the notes in one .org file?

### DISCUSSION THREAD

**Discussion by nujabse** [emacs-citar/citar#752] [Q&A] [answered]
Can citar be configured to write all the notes to one .org file instead of multiple files?

**Comment by bdarcus** [comment, marked as answer]
Yes, if you use a separate note package like `citar-org-roam` or IUC `citar-denote`.

With the former you can actually mix-and-match note-per-file and mutiple-notes-per-file.

Not sure how well optimized they are for that (specifically creating new notes), but give it a try and let us know?

**Reply by nujabse** [reply]
Thanks for the quick reply and recommendations. I'll try these packages.

**Reply by bdarcus** [reply]
No problem.

PS - I moved this to the Q&A area.
